pkg/cache/tas_cache_test.go

```diff
@@ -221,8 +222,9 @@ func TestFindTopologyAssignment(t *testing.T) {
 		tolerations    []corev1.Toleration
 		wantAssignment *kueue.TopologyAssignment
 		wantReason     string
+		leastAllocated bool
 	}{
-		"minimize the number of used racks before optimizing the number of nodes": {
+		"minimize the number of used racks before optimizing the number of nodes; LeastAllocated": {
```

Comment:
Just confirmation. After some release, we will remove the deprecated `LeastAllocatedTAS` feature gates?
In that case, could you add comments like `TODO: remove after removed LeastAllocatedTAS feature gates` to all removal test cases? Or Instead of that, we can just mention the removal cases in KEP graduation criteria.

@mimowo wdyt?

Reply:
We introduce the feature gate as a bailout mechanism as we already have users relying on TAS so we don't want to break them. 

If we don't hear complains from users for one or two releases then Im happy to drop it. I would add it to graduation criteria for GA.

Reply:
SGTM, thanks!

Reply:
> Just confirmation. After some release, we will remove the deprecated `LeastAllocatedTAS` feature gates?

Correct

Reply:
Added needed TODOs comments, mentioned dropping the flag in the KEP